Lab 10 Fixes

lmock opened the pull request:
Fixes for Lab 10

lmock committed:
Lab 10 Random Tree

Fix BYOB image to snap

lmock committed:
Lab 10 Understanding Tree Inputs

Changed byob image to snap

xtitter commented:
A few things (we can talk f2f today):
- this merge can't happen automatically, because there must be a conflict.  Can you pull from beautyjoy/bjc-r into lmock/bjc-r, figure out the conflict and fix it, and make a new pull request?  Or maybe your fixed files will autmoatically be included through github magic...  Anyway, the merging should happen on your side.  (Always remember to pull the latest from beautyjoy/bjc-r before you start editing!)
- random tree looks good -- http://lmock.github.io/bjc-r/cur/programming/recursion/tree/random-tree.html -- but it would be nice to include an xml project for that buggy code, so the students don't have to construct it (no real instructional benefit in that).  Including a new xml project is easy easy -- make it on a local snap, export to xml locally, put that xml file in /code/SOMEWHERE, and include a <A class="run" href="..."> that code in your page.
- I can't view http://lmock.github.io/bjc-r/cur/programming/recursion/tree/selftest1.html -- it won't load?

lmock committed:
Merging with Michael's changes.

lmock committed:
merging files

lmock committed:
Lab 10 Self Test Understanding Tree Inputs Fix

http://bjc.eecs.berkeley.edu/bjc-r/cur/programming/recursion/tree/selftest1.html?topic=berkeley_bjc/recur/recursion-trees-fractals.topic&step=3&novideo=true&noreading=true&course=berkeley_bjc.html

lmock committed:
Lab 10 Self Test Understanding Tree Inputs Fix

http://bjc.eecs.berkeley.edu/bjc-r/cur/programming/recursion/tree/selftest1.html?topic=berkeley_bjc/recur/recursion-trees-fractals.topic&step=3&novideo=true&noreading=true&course=berkeley_bjc.html

lmock committed:
Lab 10 Base Cases

http://bjc.eecs.berkeley.edu/bjc-r/cur/programming/recursion/tree/base-cases.html?topic=berkeley_bjc/recur/recursion-trees-fractals.topic&step=7&novideo=true&noreading=true&course=berkeley_bjc.html

Image fix

lmock committed:
Lab 10 Snowflake

http://bjc.eecs.berkeley.edu/bjc-r/cur/programming/recursion/fractals/snowflake.html?topic=berkeley_bjc/recur/recursion-trees-fractals.topic&step=8&novideo=true&noreading=true&course=berkeley_bjc.html

Fixed last byob image to snap
